tags/for.js

```diff
@@ -90,3 +92,18 @@ module.exports = function (liquid) {
     }
   })
 }
+
+function isObject(collection) {
+  if (collection) {
+    const ctr = collection.constructor
+    return ctr === Object || ctr == null
+  }
+  return false
+}
+
+function isEmpty(collection) {
```

Comment:
`isEmpty` should also capable of none-empty objects.

The underscore/lodash can be implementation references, e.g. <https://lodash.com/docs/4.17.4#isEmpty>